.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
```

Comment:
Why Travis? CircleCI is currently used for CI and I don't intend to switch to Travis (unless there are very good reasons).